src/cascadia/TerminalSettingsEditor/GlobalAppearance.xaml

```diff
@@ -28,9 +28,22 @@
 
     <ScrollViewer>
         <StackPanel Style="{StaticResource SettingsStackStyle}">
-            <!--  Theme  -->
-            <local:SettingContainer x:Uid="Globals_Theme"
+            <!--  Language  -->
+            <local:SettingContainer x:Uid="Globals_Language"

```

Comment:
I wasn't sure where I should put the ComboBox. Would it look better elsewhere? Maybe not as the first item in the Appearances section?
The reason I put it as the first item was simply because I thought it looked nicer that way.